app-web/src/components/Journey/JourneyMap.js

```diff
@@ -44,6 +44,7 @@ export const JourneyMap = ({ title, link, stops }) => {
             title
           )}
         </TopicTitle>
+        <div>{description}</div>
```

Comment:
missing props types for description. Also was the topic container using a `div` container? I don't think this is the best html tag to use.